### PR TITLE
Updated doc blocks to remove reference to bake and replaced with some…

### PR DESCRIPTION
**Pull request by garethellis36**
…thing more relevant

### CODE REVIEW

File: bin/cake.bat [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-SET lib=%~dp0
-
-php "%lib%cake.php" %*
-
-echo.
-
-exit /B %ERRORLEVEL%
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::
+:: Windows batch script for invoking CakePHP shell commands
```
Comment (markstory) [inline bin/cake.bat]: Looks like newlines may have changed. This file needs to have crlf endings as it is for windows.

**Comment by garethellis36** [comment]
Ah yes. I'll sort that.

File: bin/cake.bat (hunk 1 above)
Comment (garethellis36) [inline bin/cake.bat]: Slightly confused as when I open cake.bat in Sublime/PhpStorm, it's showing as CRLF already.

Comment (markstory) [inline bin/cake.bat]: Odd, the current file shows as CRLF for me in vim as well.

Comment (garethellis36) [inline bin/cake.bat]: I checked this out on my windows laptop and it shows as CRLF on there (and works fine). Could it be a git setting on my Mac? I could start over and do the bash script on my Mac and the .bat file on Windows to see if it helps :-)

Comment (markstory) [inline bin/cake.bat]: That might help. The line endings could be changed by mac git/editor for sure.

**Pull request closed without merging**